### PR TITLE
added platform.sh config

### PR DESCRIPTION
**Pull request by lsmith77**
we got a 2nd platform.sh instance to run the sandbox ..
can we do a final review ..
merge this and then I will ask Fabien to setup the DNS for sandbox.cmf.symfony.com

/cc @WouterJ @dbu @dantleech

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -67,6 +67,9 @@
         ]
     },
     "config": {
+        "platform": {
```
Comment (lsmith77) [inline composer.json]: this is not really related to platform.sh as we can have 5.6 there .. but I figured we keep things like this for now so that the composer.lock will not cause issues for users still on 5.4

Comment (WouterJ) [inline composer.json]: I think we should stop committing the lock file (Core SE is about to remove it again as well).

File: .gitignore [hunk 2]
```diff
@@ -9,8 +9,6 @@ web/assetic
 build/
 vendor/
 vendor
-app/config/parameters.yml
```
Comment (WouterJ) [inline .gitignore]: do we really need this? Besides being hosted, the Sandbox also serves as an example projects for people to check out. It's not that great to commit env specific information in there...

File: composer.json (hunk 1 above)
Comment (lsmith77) [inline composer.json]: that is another option of course

**Comment by lsmith77** [comment]
trying out once more if we cannot use incenteev .. but ran into some issues with the platform.sh - github - integration .. will let support sort that out next week.

File: composer.json (hunk 1 above)
Comment (lsmith77) [inline composer.json]: I will leave this decision to another PR

File: .gitignore [hunk 3]
```diff
@@ -1,22 +1,20 @@
 composer.phar
+app/cache/*
```
Comment (lsmith77) [inline .gitignore]: just sorted things a bit here
